src/openscad.cc

```diff
@@ -1044,6 +1047,21 @@ int main(int argc, char **argv)
 		arg_colorscheme = vm["colorscheme"].as<string>();
 	}
 
+	ExportFileFormatOptions exportFileFormatOptions;
+    if(vm.count("export-format")) {
+        const char *tmp_format = vm["export-format"].as<string>().c_str();
+        for(auto ff: exportFileFormatOptions.exportFileFormats) {
```

Comment:
Since exportFileFormats is a map, you can just look this up directly; no need for a loop.